cmd/root.go

```diff
@@ -12,6 +13,7 @@ import (
 )
 
 const defaultKubewardenNamespace = "kubewarden"
+const defaultPolicyServersCARootFile = "./policyservers-ca-cert"
```

Comment:
I don't think the flag should have a default value, it should be empty if the user doesn't use the `--filepath-cacert`

Reply:
Indeed, dropped the default.